examples/rosa-hcp-public-shared-vpc-resources/outputs.tf

```diff
@@ -0,0 +1,90 @@
+# output "cluster_id" {
```

Comment:
Should these be commented out?

Reply:
Oh, it is just that I don't have the cluster yet as there is no support for shared vpc attributes. It will be update in a later PR